plugin/helpers.go

```diff
@@ -61,6 +61,11 @@ type Helpers interface {
 	// Minimum server version: 5.6
 	KVListWithOptions(options ...KVListOption) ([]string, error)
 
+	// KVModify modifies data for a particular key.
+	//
+	// Minimum server version: 5.2
+	KVModify(key string, f func(initialValue []byte) ([]byte, error)) error
```

Comment:
I believe the ask was also to rename this to `KVUpdate` as per https://github.com/mattermost/mattermost-server/pull/13805#discussion_r413521175